external/model-preparation-algorithm/configs/classification/mobilenet_v3_large_1_cls_incr/hpo_config.yaml

```diff
@@ -4,12 +4,12 @@ hp_space:
   learning_parameters.learning_rate:
     param_type: qloguniform
     range:
-      - 0.0032
-      - 0.08
+      - 0.0008
+      - 0.1
       - 0.0001
   learning_parameters.batch_size:
     param_type: qloguniform
     range:
-      - 40
-      - 96
+      - 64
```

Comment:
Could you explain why only mbv3 starts from 64 bs?

Reply:
I conducted experiments for 3 cases and selected best case. So each models can have different search space.